lblod/frontend-form-builder
# SFB-216: rename the route for testing the form to view/:id/form

## ID
 [SFB-216](https://binnenland.atlassian.net/browse/SFB-216)

 ## Description

The route `/forms` is already used to get the public folder data. Renamed the route to `view` this because it will show a view of something and add `:id/form` behind it to clarify that you will view a form with id.
 ## Type of change


 ## How to test

View the `Voorbeeld` of a form. The route should be of format `BASE/view/:id/form`

 ## Links to other PR's

 - https://github.com/lblod/app-form-builder/pull/11
